OMEdit/OMEdit/OMEditGUI/Options/OptionsDialog.cpp

```diff
@@ -3360,20 +3366,42 @@ GraphicalViewsPage::GraphicalViewsPage(OptionsDialog *pOptionsDialog)
   mpDiagramViewScaleFactorSpinBox->setSingleStep(0.1);
   mpDiagramViewPreserveAspectRatioCheckBox = new QCheckBox(Helper::preserveAspectRatio);
   mpDiagramViewPreserveAspectRatioCheckBox->setChecked(true);
+  mpDiagramViewTextMinSizeLabel = new Label(Helper::textMinSize);
+  mpDiagramViewTextMinSizeSpinBox = new DoubleSpinBox;
+  mpDiagramViewTextMinSizeSpinBox->setRange(0, std::numeric_limits<double>::max());
+  mpDiagramViewTextMinSizeSpinBox->setSingleStep(0.1);
+  mpDiagramViewTextMinSizeSpinBox->setValue(9);
```

Comment:
How do you calculate this magic number?

Reply:
Do you mean `->setValue(9)`? I just replaced 12 (supposing now there is no need in so large font) with a value looking good enough. There were already (non configurable) magic numbers already (at least [here](https://github.com/OpenModelica/OpenModelica/blob/master/OMEdit/OMEdit/OMEditGUI/Annotations/TextAnnotation.cpp#L349)), I don't argue against restoring 12 or any other value. I suppose these values were guessed by experiment, too...

Reply:
Well having 12 is also not a good idea. We should use `Helper::systemFontInfo.pointSizeF()` to get the default system font size.

Reply:
Yes, it seems I mis-guessed this value. When set to 6, PID_Controller looks good:
![PID_Controller](https://user-images.githubusercontent.com/9654772/56658518-7af3a100-66a3-11e9-8950-b9a52412da26.png)

Reply:
Hmm... I have 12 by default, and `PID_Controller` looks quite ugly until I zoom it in.

Supposing `Helper::systemFontInfo.pointSizeF()` is the initial guess, should we store the absolute value or ratio with the above value? The latter may help when changing pixel density. And if the parameter would be called "_relative_ text size" (w.r.t system default), it would even be trivially implemented (like other values -- just load & store) and avoids weirdly looking numbers with many digits after the `.` when system default is changed. 

Taking into account the first statement, it may initially be set to some value less than `1.0`.

Reply:
**"relative text size" (w.r.t system default)** sounds legit.

Reply:
I get better results when its set to 2.

![image](https://user-images.githubusercontent.com/4007231/56661802-cc4f6080-66a2-11e9-8e2e-f44f20d7c382.png)

Reply:
Not sure 2 is good: look at the rightmost icon, especially, the text `10` in the center of it. 6 (that is, x0.5) looks almost identically but without such artifact.